File: ReparatieSysteem/ReparatieSysteem/Messaging/ParafiksitOrderRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ReparatieSysteem.Messaging
{
    class ParafiksitOrderRequest
    {
        public List<WorkPerformedInfo> workPerformed;
        public Contact contact;
        public ShippingAddress shippingAddress;

        public ParafiksitOrderRequest(List<WorkPerformedInfo> work)
        {
            this.workPerformed = work;
        }

        public List<WorkPerformedInfo> getWork()
        {
            return workPerformed;
        }

        public void setWork(List<WorkPerformedInfo> work)
        {
            this.workPerformed = work;
        }
    }
}

File: ParafiksitDomain/src/main/java/com/marbl/parafiksit/ParafiksitOrderRequest.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package com.marbl.parafiksit;

import com.marbl.client.domain.Contact;
import com.marbl.client.domain.ShippingAddress;
import com.marbl.client.domain.WorkPerformedInfo;
import java.util.ArrayList;
import java.util.List;

/**
 * @author Bas
 */
public class ParafiksitOrderRequest
{
    private ShippingAddress shippingAddress;
    private Contact contact;
    private List<WorkPerformedInfo> workPerformed;

    public ParafiksitOrderRequest(ArrayList<WorkPerformedInfo> work)
    {
        this.workPerformed = work;
    }

    public List<WorkPerformedInfo> getWork()
    {
        return workPerformed;
    }

    public void setWork(List<WorkPerformedInfo> work)
    {
        this.workPerformed = work;
    }

    public ShippingAddress getShippingAddress()
    {
        return shippingAddress;
    }

    public Contact getContact()
    {
        return contact;
    }

    public List<WorkPerformedInfo> getWorkPerformed()
    {
        return workPerformed;
    }
    
    
    
}
